### ISSUE TITLE
Questions about reproducing result

### ISSUE THREAD

**Issue by rexdu2003** [SivanDoveh/TSVLC#8]
Dear Authors:

Thanks for the great work! I appreciate the idea that using explicit negative text examples to enhance the ability of SVLC.

However, I have two problems on reproducing the results with your codes:
1. Regarding the checkpoint in the link [https://drive.google.com/drive/folders/1WosT_kdam1ymWjVSK2ezyydLoqmm0LdX?usp=sharing](https://drive.google.com/drive/folders/1WosT_kdam1ymWjVSK2ezyydLoqmm0LdX?usp=sharing) you provided in `README.md`, it is said that the checkpoint is trained with [LLM,RB] negatives. However, I found that the results are more similar to those of rule-based negatives as shown in [out.log](https://github.com/SivanDoveh/TSVLC/files/13545304/out.log). 
2. I use the batch size of 512 with one A100, instead of 4 V100 as stated in the paper. I keep the other settings unchanged. However, with using only rule-based negatives trained with ViT-B/32 finetuning for 5 epochs, I found that I get a much lower result. I wonder whether it is because the large variance in the performance or there is something run in my hyperparameter choices. My training log is [out1.log](https://github.com/SivanDoveh/TSVLC/files/13545357/out1.log). It will be helpful if you could share the training log so that I can check my experimental settings.

Thanks for taking time reading my words!

**Comment by SivanDoveh** [comment]
Thanks, it's always fun to see that people read and use it :)

1. There are two checkpoints I added to the drive- one for the [LLM, RB] and one for the RB negatives training.
2. You should get the same results (but with longer training time). What are the results you're getting that are low, VL-CHECKLIST? Can you share your results?

**Comment by nmndeep** [comment]
Hi,
Cool work!

We tried to recreate the results with your fine-tuning, but struggle quite a lot. For instance, we can only get ~49% Imagenet accuracy and here is the associated log. We ran it for 10 epochs with a BS=150x4.
Could you have a look as to what we might be doing wrong there. 

[out.log](https://github.com/user-attachments/files/19483270/out.log)

Best,
Naman
